src/status_im2/contexts/syncing/sheets/enter_password/views.cljs

```diff
@@ -0,0 +1,60 @@
+(ns  status-im2.contexts.syncing.sheets.enter-password.views
+  (:require [quo.react-native :as rn]
+            [utils.re-frame :as re-frame]
+            [quo2.core :as quo2]
+            [quo.core :as quo]
+            [i18n.i18n :as i18n]
+            [clojure.string :as string]
+            [quo2.foundations.colors :as colors]
+            [status-im.constants :as constants]
+            [status-im.ui.components.qr-code-viewer.views :as qr-code-viewer]))
+
+(defn qr-code-view-with-connection-string [connection-string]
+  (let [window-width                (re-frame/sub [:dimensions/window-width])
+        eighty-percent-screen-width (* window-width 0.8)
+        valid-cs?                   (string/starts-with? connection-string constants/local-pairing-connection-string-identifier)]
+    [:<>
+     (if valid-cs?
+       [rn/view {:margin 20}
+        [quo2/text {:accessibility-label :sync-code-generated
+                    :weight              :bold
+                    :size                :heading-1
+                    :style               {:color colors/neutral-100
+                                          :margin 20}}
+         (i18n/label :t/sync-code-generated)]
+        [qr-code-viewer/qr-code-view eighty-percent-screen-width connection-string]
+        [quo2/information-box {:type      :informative
+                               :closable? false
+                               :icon      :i/placeholder
+                               :style     {:margin-top 20}} (i18n/label :t/instruction-after-qr-generated)]]
+       [rn/view {:margin 20}
+        [rn/view {:padding-horizontal 8}
+         [quo/button
+          {:on-press #(re-frame/dispatch [:preparations-for-connection-string])}
+          (i18n/label :t/try-your-luck-again)]]])]))
+
+(defn views []
```

Comment:
enter-password

Reply:
@flexsurfer : are you asking me to rename the views method to enter-password? or asking me to rename folder from enter_password to enter-password?

Reply:
method :)

Reply:
so views.cljs saying that this namespace has views, not we should have views properly named in this namespace, so it's a views namespace with one enter-password view

Reply:
btw it should view, not views